GLOSSARY.md

```diff
@@ -0,0 +1,21 @@
+## Glossary
+
+[Early draft, please add new definitions, fix the existing ones, add links and correct organizational details!]
+
+* **AFDKO** [Adobe Font Development Kit for OpenType](http://adobe-type-tools.github.io/afdko/AFDKO-Overview.html).
+* **AHG** ad-hoc group, a group of people representing multiple stakeholders, who meet informally to discuss technical proposals (example: the AHG who got together in 2015–2016 and made OpenType Variations happen), perhaps bypassing corporate policies on formal meetings.
+* **CG** Community Group, in a font technology context probably the W3C Font and Text Community Group established 2020.
+* **FEA** OpenType Feature File (extension .fea), [afdko: OpenType™ Feature File Specification](http://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html).
+* **FEE** Font Engineering language with Extensibility, a language for creating font feature code that can be compiled to FEA using [fontFeatures](https://github.com/simoncozens/fontFeatures/blob/master/docs/fee-format.rst).
```

Comment:
Is that the authoritative URL for FEE? I find it slightly surprising to have a link to a personal page.

Reply:
I have to imagine someday FEE would be split out into an independent project, but given that the personal page in question is that of the inventor and implementer I think the surprise factor should be pretty small. I guess we can ask @simoncozens if he has any other URL that would be better to document what FEE is, but that was the best one I could find.

Reply:
No, I think that's the best URL. If it ever moves, we can change the glossary...